### ISSUE TITLE
isSet not created in some cases of modifiable

### ISSUE THREAD

**Issue by ldriscoll** [immutables/immutables#544]
Using 2.4.2 Given the following mapping:
```java
@Value.Immutable
@Value.Modifiable
@Value.Style(
    get = {"is*", "get*"},
    init = "set*",
    typeAbstract = {"Abstract*"}, 
    typeImmutable = "*",
    optionalAcceptNullable = true,
    forceJacksonPropertyNames = false,
    validationMethod = ValidationMethod.VALIDATION_API,
    visibility = ImplementationVisibility.SAME)
public abstract class AbstractImmutableWithModifiable {
  int getId();
  String getDescription();
  List<String> getNames();
}
```
the `ModifiableImmutableWithModifiable` class is generated with a toString() like this:
```java
  @Override
  public String toString() {
    return "ModifiableImmutableWithModifiable{"
        + "id="  + (idIsSet() ? getId() : "?")
        + ", description=" + getDescription()
        + ", names=" + getNames()
        + "}";
  }
```

but the `idIsSet()` method does not exist.  However, if I remove the `validationMethod = ValidationMethod.VALIDATION_API,` the method is created

**Comment by elucash** [comment]
It appears to be fixed in master and will be released soon in 2.4.3

**Comment by elucash** [comment]
duplicate of #536, released in 2.4.3. Thank you!
